### PR TITLE
docs: docstrings jaml

### PR DESCRIPTION
**Pull request by cristianmtr**

### CODE REVIEW

**Review by hanxiao — APPROVED** [review]

**Review by hanxiao — CHANGES_REQUESTED** [review]
broken test

File: jina/jaml/__init__.py [hunk 1]
```diff
@@ -90,6 +94,9 @@ def load(stream,
     @staticmethod
     def load_no_tags(stream, **kwargs):
         """Load yaml object but ignore all customized tags, e.g. !Executor, !Driver, !Flow
+        :param stream: the output stream
+        :param **kwargs: other kwargs
+        :return: the class
```
Comment (JoanFM) [inline jina/jaml/__init__.py:99]: return the instantiated object, not the class

**Review by theUnkownName — DISMISSED** [review]

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Comment by cristianmtr** [comment]
> broken test

Which one was it?

**Comment by cristianmtr** [comment]
> LGTM

Thanks for helping, @nan-wang

File: jina/jaml/helper.py [hunk 2]
```diff
@@ -87,8 +87,20 @@ def parse_config_source(path: Union[str, TextIO, Dict],
                         allow_dict: bool = True,
                         allow_json: bool = True,
                         *args, **kwargs) -> Tuple[TextIO, Optional[str]]:
-    """ Check if the text or text stream is valid
-
+    """Check if the text or text stream is valid.
+
+    # noqa: DAR401
+    :param path: the multi-kind source of the configs.
+    :param allow_stream: flag
```
Comment (JoanFM) [inline jina/jaml/helper.py:94]: it would be nice to explain better these params, let me see if I can provide some better definitions in a while

**Review by theUnkownName — APPROVED** [review]
LGTM!

File: jina/jaml/helper.py [hunk 3]
```diff
@@ -87,8 +87,20 @@ def parse_config_source(path: Union[str, TextIO, Dict],
                         allow_dict: bool = True,
                         allow_json: bool = True,
                         *args, **kwargs) -> Tuple[TextIO, Optional[str]]:
-    """ Check if the text or text stream is valid
-
+    """Check if the text or text stream is valid.
+
+    # noqa: DAR401
+    :param path: the multi-kind source of the configs.
```
Comment (JoanFM) [inline jina/jaml/helper.py:93]: I do not see how to add suggestions now?

Comment (cristianmtr) [reply jina/jaml/helper.py:93]: I merged it after receiving two approvals

Comment (cristianmtr) [reply jina/jaml/helper.py:93]: You can make a new PR

**Pull request merged**